### PR TITLE
Fix ubuntu_usn importer for invalid CVE

### PR DESCRIPTION
**Pull request by AmitGupta7580**
Error:

image

Fixed:

image

### CODE REVIEW

File: vulnerabilities/importers/ubuntu_usn.py [hunk 1]
```diff
@@ -74,7 +75,7 @@ def to_advisories(usn_db):
                 # The db sometimes contains entries like
                 # {'cves': ['python-pgsql vulnerabilities', 'CVE-2006-2313', 'CVE-2006-2314']}
                 # This `if` filters entries like 'python-pgsql vulnerabilities'
-                if not cve.startswith("CVE-"):
+                if not is_cve(cve):
                     continue
```
Comment (sbs2001) [inline vulnerabilities/importers/ubuntu_usn.py:79]: skipping entirely would be bad idea as we'll be losing other data. 

Instead I suggest 

```
  if not is_cve(cve):
                    cve = ""
```

**Review by sbs2001 — CHANGES_REQUESTED** [review]

File: vulnerabilities/importers/ubuntu_usn.py (hunk 1 above)
Comment (AmitGupta7580) [reply vulnerabilities/importers/ubuntu_usn.py:79]: Yes, It's a good idea as empty CVE will automatically takes the VULCOID-X-X-X.

**Comment by AmitGupta7580** [comment]
Around 600-700 vulnerabilities increased now.

image

image

**Review by sbs2001 — APPROVED** [review]
Thanks @AmitGupta7580  I'm merging this

**Pull request merged**